modules/user_accounts/templates/form_edit_user.tpl

```diff
@@ -266,9 +265,11 @@
     	</div>
     </div>
     {/if}
-
-    {if $form.errors.Email|default}
+    {if $form.errors.Email_Group|default}
```

Comment:
I don't think this is the correct solution, you are just undoing work from #7274 I think what should be done instead is to replace email_group in the edit_user class by "email" in the validate function.